content/blog/2021/cfp-announcement.md

```diff
@@ -0,0 +1,99 @@
+
+
+### Speaker Info
+
+You'll be asked to enter your biography and the information will be made public. Here, you may wish to include more information for reviewers to complement information about your previous experience. This can include your subject matter experience, past workshops if any, past speaking events if any, or anything else that would help us understand and gauge your abilities as a workshop presenter. A link to a video would be ideal (to be added in speaker URLs)
+
+
+## Guidelines for creating Preview Video
+
+With PyCon India 2021 being online, we are reintroducing preview videos to our proposal submission workflow. **Preview videos** are you talking about your proposal, topics you intend to cover in the talk, and how you intend to cover them. Participants are strongly suggested to upload links to their preview videos while submitting their proposals. It will be immensely helpful for reviewers to go through preview videos and take that into account before making final decisions on your talk proposal. Please keep in mind to strictly follow all guidelines for creating your video preview as mentioned below:-
+
+
+1. **Duration & orientation**  
+A 1-3 minute video to be recorded, preferably in portrait mode.
+
+
+2. **Upload to YouTube**  
+The video has to be uploaded on Youtube. If you don’t want other people to find it, mark it as unlisted. Don’t mark it as private or disallow embedding, or we won’t be able to see it. ​
+
+
+3. **Only you talking about your talk**  
+The video should contain nothing except you talking about your talk.​ Try to make a video that holds our attention and helps us understand more deeply about what your talk is all about.
+
+
+4. **No effects & No Music**  
+Please do not add any background music to your video. ​No screenshots or postproduction wizardry please; we don’t want this to turn into a video-making contest. If you’re going to spend time making something cool, put that into your slides & proposal instead.
+
+
+5. **No Script**  
+Please do not recite a script written beforehand. Just talk spontaneously as you would to a friend. People delivering memorized speeches (or worse still, reading text off the screen) usually come off as dull and uninspiring. Think of this as a fun activity and let your heart do the talking. Be vanilla!
+
+
+6. **Check your Audio**  
+Try to keep your voice clear and check that it's being recorded properly. Make sure there isn't any background noise. Try to record this in an empty room if possible without much going on in the background.
+
+## Have any more questions? We are here to help!
+
+Feel free to ping us on [Zulip](https://pyconindia.zulipchat.com/) or IRC (#pyconindia) or contact the coordinators through email:
+
+-

```

Comment:
Please add the contact details point(s) of contact for CFP questions here.

Reply:
cfp@in.pycon.org

Reply:
It's in the next line.